### PR TITLE
Wren

### PR DESCRIPTION
**Pull request by marghidanu**
## Description

Wren language implementation

## Contributing requirements

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeWren/solution_1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+RUN apt-get update \
+    && apt-get install -y curl busybox 
+
+RUN curl -sL https://github.com/wren-lang/wren-cli/releases/download/0.3.0/wren_cli-linux-0.3.0.zip | busybox unzip - -d / \
+    && cp /wren_cli-linux-0.3.0/wren_cli /usr/local/bin \
+    && rm -rf /wren_cli-linux-0.3.0 \
+    && chmod a+x /usr/local/bin/wren_cli 
+
+WORKDIR /opt/app
+COPY *.wren .
```
Comment (rbergen) [inline PrimeWren/solution_1/Dockerfile]: I'm thinking that it may be better to add a slash to the target, considering that it's a wildcard copy?

Comment (marghidanu) [reply PrimeWren/solution_1/Dockerfile]: Should be ok now

Comment (rbergen) [reply PrimeWren/solution_1/Dockerfile]: Yep, that's even cleaner. :)

**Review by rbergen — APPROVED** [review]
LGTM!

**Pull request merged**